<?xml version="1.0"?>
<launch>
  <arg name="chain_start" default="base_link" />
  <arg name="chain_end" default="moveo_nozzle_link" />
  <arg name="timeout" default="0.2" />
  <arg name="timeout_second" default="0.05" />
  <arg name="eps" default="1e-5"/>
  <!--
    arg name="num_threads" default="16"/
    -->
  <arg name="model" default="$(find moveo_urdf)/urdf/moveo_urdf.urdf"/>
  <param name="robot_description" textfile="$(arg model)" />
  
  <arg name="gcode_out" default="$(find moveo_moveit_move_group)/area/all_area.csv"/>
  <arg name="node_start_delay" default="1.0" />  
  <node name="area_test" pkg="moveo_moveit_move_group" type="area_test" output="screen">
    <param name="chain_start" value="$(arg chain_start)"/>
    <param name="chain_end" value="$(arg chain_end)"/>
    <param name="timeout" value="$(arg timeout)"/>
    <param name="timeout_second" value="$(arg timeout_second)"/>
    <param name="eps" value="$(arg eps)" type = "double"/>
    <!--
      param name="num_threads" value="$(arg num_threads)" type = "int"/
      -->
    <param name="urdf_param" value="/robot_description"/>
    <param name="gcode_out" value="$(arg gcode_out)"/>
  </node>
</launch>
